include/SDL_config.h.cmake

```diff
@@ -318,6 +318,7 @@
 #cmakedefine SDL_JOYSTICK_HIDAPI @SDL_JOYSTICK_HIDAPI@
 #cmakedefine SDL_JOYSTICK_EMSCRIPTEN @SDL_JOYSTICK_EMSCRIPTEN@
 #cmakedefine SDL_JOYSTICK_VIRTUAL @SDL_JOYSTICK_VIRTUAL@
+#cmakedefine SDL_JOYSTICK_TRANSLATOR @SDL_JOYSTICK_TRANSLATOR@
```

Comment:
- SDL_config.h.in needs the same addition.
- Other SDL_config_XXX.h probably need the addition,   enabled or
  disabled same way SDL_JOYSTICK_VIRTUAL is enabled in them.